tests/test_launch_methods/test_jsrun/test_1/res_set

```diff
@@ -0,0 +1 @@
+RS 0: { host: 1, cpu: 0}
```

Comment:
Should we use ERF?

Reply:
Yes I agree. @iparask let me know how we can proceed on that.

Reply:
@vivek-bala I'll make a commit now that you are done